indexer/src/indexer.ts

```diff
@@ -744,6 +737,48 @@ export const createIndexerService = (config: IndexerDbCredentials) => {
                             WHERE i.timestamp >= NOW() - (${intervalSql} || ' days')::INTERVAL
                               AND plt.pool = ${poolAddressesSql}
                             GROUP BY plt.pool, plt.lp_token),
+             CurrentIncentives AS (SELECT i.lp_token,
```

Comment:
Can we return the different ongoing incentives aswell for the incentive tooltip. We display each ongoing incentive in the frontend